### DISCUSSION TITLE
Are deployed sites from v6 compatible with v7?

### DISCUSSION THREAD

**Discussion by thinkcube-solutions** [deployphp/deployer#3424] [General]
Hello,

Just a general question on upgrading.

I am upgrading some sites that are currently deployed with v6. If I upgrade to Deployer 7 and update the deploy.php configuration file, will the structure that is already deployed on a server with v6 be compatible as is with v7?

I wasn't sure if a completely fresh instance for a v7 structure was needed or if it's a smooth transition or not.

Thank you!

**Comment by antonmedv** [comment]
There is a doc on how to migrare.

**Reply by antonmedv** [reply]
Migrate
